### PR TITLE
samples(spanner): add JSON samples

### PR DESCRIPTION
**Pull request by larkee**

### CODE REVIEW

**Review by zoercai — APPROVED** [review]

**Comment by larkee** [comment]
@bshaffer Could you please approve and merge this PR? :)

File: spanner/src/add_json_column.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * @param string $databaseId The Spanner database ID.
+ */
+function add_json_column($instanceId, $databaseId)
+{
+    $spanner = new SpannerClient();
+    $instance = $spanner->instance($instanceId);
+    $database = $instance->database($databaseId);
+
+    $operation = $database->updateDdl(
+        "ALTER TABLE Venues ADD COLUMN VenueDetails JSON"
```
Comment (bshaffer) [inline spanner/src/add_json_column.php:46]: Use single quotes unless double quotes are needed for string interpolation!

File: spanner/src/query_data_with_json_parameter.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+ */
+function query_data_with_json_parameter($instanceId, $databaseId)
+{
+    $spanner = new SpannerClient();
+    $instance = $spanner->instance($instanceId);
+    $database = $instance->database($databaseId);
+
+    $exampleJson = [
+        "rating" => 9,
+        "open" => true,
```
Comment (bshaffer) [inline spanner/src/query_data_with_json_parameter.php:48]: Same with these two lines

File: spanner/src/update_data_with_json_column.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                    "[{\"name\":\"room 1\",\"open\":true},{\"name\":\"room 2\",\"open\":false}]"
+            ],
+            [
+                'VenueId' => 19,
+                'VenueDetails' => "{\"rating\":9,\"open\":true}"
+            ],
+            [
+                'VenueId' => 42,
+                'VenueDetails' =>
+                    "{\"name\":null,\"open\":{\"Monday\":true,\"Tuesday\":false},\"tags\":[\"large\",\"airy\"]}"
```
Comment (bshaffer) [inline spanner/src/update_data_with_json_column.php:63]: Using single quotes here will make this much more readable

**Review by bshaffer — COMMENTED** [review]
This looks great! Just a few style items and it's good to go

File: spanner/src/add_json_column.php (hunk 1 above)
Comment (larkee) [reply spanner/src/add_json_column.php]: Ah, good to know! Thanks :)

**Comment by larkee** [comment]
> This looks great! Just a few style items and it's good to go

Thanks for the quick response! Fixed the style issues :)

**Comment by larkee** [comment]
@bshaffer PTAL :)

**Review by bshaffer — APPROVED** [review]

**Pull request merged**